tsdb/engine/tsm1/mmap_windows.go

```diff
@@ -0,0 +1,11 @@
+// portability wrapper for Mmap/Munmap
+// +build windows
+
+package tsm1
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+var Mmap = windows.Mmap
```

Comment:
I can't find this in the package?  I don't think this will even compile on windows.  I'll confirm this today.